packages/manager/modules/pci/src/projects/project/data-integration/tabs/dashboard/components/jobsList/style.scss

```diff
@@ -0,0 +1,5 @@
+pci-project-data-integration-jobs-list {
+  .oui-pagination {
+    margin-top: 10px;
+  }
```

Comment:
Can you please avoid overriding the styles specific for few screens? It doesn't provide the same user experience to the users across the Manager.

Reply:
Hi @anooparveti , this was requested, as the `oui-tile` component containing the table seems to overwrite default margin to 0, which does not look good.
![image](https://github.com/ovh/manager/assets/24541481/96a39323-ad55-4ab5-8bbb-d7be8aa88569)

I have checked some datagrids that are on the manager, and they do have a margin on top of pagination: 
![image](https://github.com/ovh/manager/assets/24541481/d2749538-2d36-48d2-b698-f2288c6c5236)

I think it makes sense to include a margin for this specific table where a margin above the pagination is requested. 
However, maybe it should be handled in the `ovh-ui-kit`. I can declare a bug in the project, and once it is solved, we would be able to remove this custom css here.